nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkComponent.java

```diff
@@ -50,6 +51,12 @@
         return null;
     }
 
+    @Override
+    public int getApplicationStartedOrder() {
+        // Need to be started before the StreamService because our processor need the BulkAdminService
```

Comment:
need -> needs

And maybe explain that the StreamService is started after the KafkaConfigService, so relying on its order allows to start before the StreamService?

In fact, not quite sure to get the dependencies between all services: BulkComponent, BulkAdminService, StreamService, KafkaConfigService, ... Maybe a short description could help, if it doesn't exist.